packages/lesswrong/components/sunshineDashboard/NewModeratorActionDialog.tsx

```diff
@@ -17,7 +17,7 @@ const NewModeratorActionDialog = ({classes, onClose, userId}: {
   const { WrappedSmartForm, LWDialog } = Components;
   
   return (
-    <LWDialog open={true}>
+    <LWDialog open={true} onClose={onClose}>
```

Comment:
This was the one case where removing the external clickaway required changing something in a dialog component. All the others already had both the external `LWClickAwayCancel` and this `onClose` which would independently close the dialog